SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+qBittorrent takes the security of our software seriously, including all source code repositories managed through our GitHub organisation.
+If you believe you have found a security vulnerability in qBittorrent, please report it to us as described below.
+
+## Reporting Security Issues
+
+Please do not report security vulnerabilities through public GitHub issues. Instead, please use GitHubs private vulnerability reporting functionality associated to this repository. Additionally, you may email us with all security-related inquiries and notifications at `security@qbittorrent.org`.
```

Comment:
>GitHubs private vulnerability reporting functionality associated to this repository

Is it enabled? Where can I find it?

Reply:
I believe it needs to be enabled and made the assumption that it would be used when writing this up :)

Reply:
@sledgehammer999 
Could you adjust this repo settings? You can follow https://github.com/qbittorrent/docker-qbittorrent-nox/settings/security_analysis as a reference.